hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CalcRemovePhysicalRule.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.opt.physical;
+
+import com.hazelcast.jet.sql.impl.opt.physical.CalcRemovePhysicalRule.Config;
+import org.apache.calcite.plan.RelOptRuleCall;
+import org.apache.calcite.plan.RelRule;
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.rules.SubstitutionRule;
+import org.immutables.value.Value;
+
+// Copied from CalcRemovePhysicalRule.
+// Removes trivial Calc physical relation.
+@Value.Enclosing
+public class CalcRemovePhysicalRule extends RelRule<Config> implements SubstitutionRule {
```

Comment:
Why do we need this rule in physical opt?

Reply:
It's missing physical opt rule counterpart.

Reply:
Why there should be a counterpart?

Reply:
Because during the **physical** optimization phase there might be a situation when calc is trivial and may (should) be removed.